redash/handlers/authentication.py

```diff
@@ -106,8 +120,7 @@ def login(org_slug=None):
     elif current_org == None:
         return redirect('/')
 
-    index_url = url_for("redash.index", org_slug=org_slug)
-    next_path = request.args.get('next', index_url)
+    next_path = get_next_path(org_slug)
```

Comment:
We use the same pattern of next_path in other places (`redash.google_oauth` and `redash.ldap_auth`) -- mind adding there was well?

Reply:
@arikfr I have checked google oauth login with invalid `next` parameter works correctly. Not only by browsing but also by debugging with pdb.

When called `/oauth/google` in
https://github.com/getredash/redash/blob/v5.0.1/redash/authentication/google_oauth.py#L69
`next` parameter is already sanitized.

By via this code:
https://github.com/getredash/redash/blob/v5.0.1/redash/handlers/authentication.py#L131

I suppose LDAP login is the same, but I haven't checked it because I don't have the environment. Sorry.

Reply:
You mean it's sanitized because we're redirected there from `/login`? But an attacker can just send you to `/oauth/google` directly with an evil `next` URL.

I don't have a setup for LDAP either :) But I think it's ok to blindly apply your `get_next_path` function there instead of taking the value as is.

Reply:
OK. I'll fix it.